leshan-core/src/main/java/org/eclipse/leshan/json/JsonArrayEntry.java

```diff
@@ -132,30 +132,15 @@ public boolean equals(Object obj) {
         if (getClass() != obj.getClass())
             return false;
```

Comment:
This equals methods is generated code. I'm not sure we want to modify/simplify it ?